README.md

```diff
@@ -38,7 +38,7 @@ logger.commons.level = debug
 
 ## Code of Conduct
 
-This project has adopted an [Open Source Code of Conduct](https://opendistro.github.io/for-elasticsearch/codeofconduct.html).
+This project has adopted an [Open Source Code of Conduct](https://github.com/opensearch-project/OpenSearch/blob/main/CODE_OF_CONDUCT.md).
```

Comment:
Here is also a page of "Code of Conduct": https://www.opensearch.org/codeofconduct.html, but either is fine. 😁

Reply:
I actually updated this section to how it is mentioned in [CONTRIBUTING.md](https://github.com/opensearch-project/common-utils/blob/main/CONTRIBUTING.md#code-of-conduct) of this repository to keep it consistent.

Reply:
Ah, sounds like a new idea 👍

Reply:
While ours may be identical to that (I think), we tend to keep a copy to avoid external links changing on us.